### DISCUSSION TITLE
Twitter ads

### DISCUSSION THREAD

**Discussion by Mohammed1520** [ReVanced/revanced-discussions#359] [General] [answered]
Is there any tutorial about patching twitter with timeline ads patch

**Comment by oSumAtrIX** [comment, marked as answer]
Follow [the documentation](https://github.com/revanced/revanced-documentation).

**Comment by lotusdroid** [comment]
Prerequire: Zulu JDK 17
Download all of these files and rename it to what I show in this screenshot
![image](https://user-images.githubusercontent.com/106517171/180611092-f2468909-2a49-4fb3-9d4d-a4d88eb116bd.png)
Run this command:
java -jar revanced-cli-all.jar -a twitter.apk -o retwitter.apk -b revanced-patches.jar

**Comment by XRedCubeX** [comment]
sorry for the ignorance but what does twitter have to do with it? do you have an example?

**Reply by TheJeterLP** [reply]
You can patch twitter or any other app with revanced too. Its not just youtube and youtube music.

**Reply by XRedCubeX** [reply]
> You can patch twitter or any other app with revanced too. Its not just youtube and youtube music.

Oh yeah, so the other apps have a built-in youtube client?

**Reply by TheJeterLP** [reply]
no. This is not about youtube ads.

**Reply by TheJeterLP** [reply]
You can modify any app however you want with revanced. This patch removes ads in the twitter timeline.

**Reply by XRedCubeX** [reply]
> no. This is not about youtube ads.

OK thanks for the info
